frontend/src/component/strategies/StrategyView/StrategyDetails/StrategyDetails.tsx

```diff
@@ -84,7 +84,7 @@ export const StrategyDetails = ({
                 </Grid>
 
                 <Grid item sm={12} md={toggles.length > 0 ? 6 : 12}>
-                    <h6>Applications using this strategy</h6>
+                    <h6>Applications using this strategy {applications.length >= 1000 && '(Capped at 1000)'}</h6>
```

Comment:
Is equal needed?

Reply:
we get 1000 max so we don't know if there's more. So If I get max then I'm assuming there may be more and add the info